yt_dlp/extractor/zaiko.py

```diff
@@ -90,3 +92,27 @@ def _real_extract(self, url):
                 'thumbnail': ('poster_url', {url_or_none}),
             })),
         }
+
+
+class ZaikoETicketIE(ZaikoBaseIE):
+    _VALID_URL = r'https?://zaiko\.io/account/eticket/(?P<id>\w{48}=)'
```

Comment:
support `www` subdomain because it works in browser
are we certain the ticket ID is always 48 chars? would it hurt to be more lenient?
```suggestion
    _VALID_URL = r'https?://(?:www.\)?zaiko\.io/account/eticket/(?P<id>\w{48}=)'
```

Reply:
> support `www` subdomain because it works in browser

Done.

> are we certain the ticket ID is always 48 chars? would it hurt to be more lenient?

Well, it could be more lenient, but I don't know if Zaiko will remove the trailing equal sign after lengthening the ticket ID. So, let's wait for future bug reports.

Reply:
Found some actual example URLs in the wild. The IDs:

```
TZjYzNjM1YmIxZHwyMDIyMDIyMjE1MDcxNXwxMzVlZjhlNGI=
TNGE0MDJjMjYxYnwyMDIxMDkyMTExMTkwOXxmNWU4ZGVlMg==
```

looks like the character count checks out. Also looks like urlsafe b64 encoding, with an extra trailing `=`

I think we should also match `-` and `=` within the id

Reply:
> EDIT: actually it's the letter `T` followed by base64 (credit: Grub4K for figuring that out)

Oh, now all ticket IDs can be decoded to readable strings.

Should I replace the regex with "```r'https?://(?:www.)?zaiko\.io/account/eticket/(?P<id>T[\w=-]{48})'```"?

Reply:
we don't know that the first character will always be `T`, so I think we could leave the `_VALID_URL` as-is

but we could do something like this:
```py
        ticket_id = self._match_id(url)
        ticket_id = try_call(
            lambda: base64.urlsafe_b64decode(ticket_id[1:]).decode().replace('|', '-')) or ticket_id
```